lib/afterpay/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Afterpay
-  VERSION = "0.1.8".freeze
+  VERSION = "0.1.8"
```

Comment:
should this be a different version # - such as 2.0 since that will correspond to the Afterpay API version #?